MediaBrowser.Providers/Music/MusicBrainzAlbumProvider.cs

```diff
@@ -714,34 +714,32 @@ private string GetFirstReleaseGroupId(XmlReader reader)
             return null;
         }
 
-        internal Task<HttpResponseInfo> GetMusicBrainzResponse(string url, bool isSearch, CancellationToken cancellationToken)
-        {
-            return GetMusicBrainzResponse(url, isSearch, false, cancellationToken);
-        }
-
         /// <summary>
-        /// Gets the music brainz response.
+        /// Makes request to MusicBrainz server and awaits a response.
         /// </summary>
-        internal async Task<HttpResponseInfo> GetMusicBrainzResponse(string url, bool isSearch, bool forceMusicBrainzProper, CancellationToken cancellationToken)
+        internal async Task<HttpResponseInfo> GetMusicBrainzResponse(string url, CancellationToken cancellationToken)
         {
-            var urlInfo = new MbzUrl(MusicBrainzBaseUrl, 1000);
-            var throttleMs = urlInfo.throttleMs;
+            // The Jellyfin user-agent is unrestricted but source IP must not exceed
+            // one request per second, therefore we rate limit to avoid throttling
+            // https://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting
+            const long QueryIntervalMs = 1000u;
 
-            if (throttleMs > 0)
+            // Only delay if necessary
+            if (_stopWatchMusicBrainz.ElapsedMilliseconds < QueryIntervalMs)
             {
-                // MusicBrainz is extremely adamant about limiting to one request per second
```

Comment:
worth keeping the comment IMHO

Reply:
I gave a fully comment slightly above this old entry..

Reply:
Added this comment back now the interval has moved up the page.